### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by zamentur**
## Problem
- *This version fix :*
- *the corrupt sources issues.*
- *the miss of transparency about change made on upstream, here I have found a fork from someone who did the imgur deactivation... He have just missed some  label change and a google analytics script.*

## Solution
- *And how you fix that ?*

## PR Status
- [x] Code finished.
- [x] Tested with Package_check.
- [x] Fix or enhancement tested.
- [ ] Upgrade from last version tested.
- [x] Can be reviewed and tested.

## Validation
---
*Minor decision*
- **Upgrade previous version** : 
- [x] **Code review** : Maniack C
- [x] **Approval (LGTM)** : Maniack C
- [ ] **Approval (LGTM)** : 
- **CI succeeded** : 
[Build Status/badge/icon)](https://ci-apps-dev.yunohost.org/jenkins/job/strut_ynh%20testing%20(Official)/)
When the PR is marked as ready to merge, you have to wait for 3 days before really merging it.

### CODE REVIEW

**Comment by maniackcrudelis** [comment]
And, what about our testing process ?

**Comment by JimboJoe** [comment]
CI failed, I relaunched it...

**Comment by zamentur** [comment]
@maniackcrudelis So what's the process now it pass all tests ?

**Comment by frju365** [comment]
@zamentur : now we have to tick every review checkboxes.

I had a question : why upgrade commit has been removed ? Because, like that, Upgrade have to be checked manually, doesn't it ?

**Comment by zamentur** [comment]
We can't test if upgrade from this previous version will be ok because this previous is no longer installable (Corrupt source).

This pr change the upstream for a fork that respect privacy (it's not the case of the unmaintained strut project)

**Comment by zamentur** [comment]
Bop bidi bop bop bop

**Comment by zamentur** [comment]
I appeal to the light validation https://github.com/YunoHost/project-organization#validation-all%C3%A9g%C3%A9-en-cas-de-manque-de-relecteurs

**Comment by JimboJoe** [comment]
I think we're missing a huge step here: testing versions must be advertised on the official forum topic for one week before being reviewed/merged.
Could you please create the Strut official topic on the forum (and add it [here](https://forum.yunohost.org/t/index-of-official-apps-threads/4127))?
Thanks! :wink:

**Comment by zamentur** [comment]
Done https://forum.yunohost.org/t/official-app-strut/5788

**Comment by maniackcrudelis** [comment]
Ready to be reviewed

File: scripts/install [hunk 1]
```diff
@@ -47,7 +47,10 @@ ynh_clean_setup () {
     ynh_secure_remove "$tmp_dir"
 }
 ynh_setup_source "$tmp_dir"
-mv "$tmp_dir/sources"  "$final_path"
+mv "$tmp_dir/dist"  "$final_path"
```
Comment (maniackcrudelis) [inline scripts/install]: What about the other directories ?
Is `dist` the only interesting directory in this archive ?

Comment (zamentur) [reply scripts/install]: Dist is the compiled version. Other directories are source code and tools to build the program.

**Comment by zamentur** [comment]
Done

**Comment by maniackcrudelis** [comment]
Code review OK

**Pull request merged**